src/vs/editor/contrib/rtv/RTVSynthService.ts

```diff
@@ -0,0 +1,327 @@
+import { TableElement, isHtmlEscape } from 'vs/editor/contrib/rtv/RTVUtils';
+
+export class RTVSynthService {
```

Comment:
Should we rename it to something like `RTVSynthModel`? Since it essentially acts as the model, right?